src/backend/cdb/cdbvars.c

```diff
@@ -277,6 +277,7 @@ int			gp_hashagg_groups_per_bucket = 5;
 int			gp_motion_slice_noop = 0;
 
 /* Cloudberry Database Experimental Feature GUCs */
+bool		gp_enable_explain_rows_out = false;
```

Comment:
It is better  to send a param to explain command than using a guc to control if print out the "rows out"

Reply:
Rows out feature created for auto_explain. If we add "rows out" as parameter of explain, then we'll need change auto_explain source, and auto_explain wont be compatible with vanilla

Reply:
```
 auto_explain wont be compatible with vanilla
```
Hmm... cannot understand. Could you give more details? Is auto_explain aslo include in vanilla?

Reply:
auto_explain is included in vanilla https://github.com/greenplum-db/gpdb-archive/tree/main/contrib/auto_explain

Also GUC gp_enable_explain_rows_out is made by analogy with GUC gp_enable_explain_allstat